cpp/cograph_recognition/cograph/Recognition.cpp

```diff
@@ -0,0 +1,549 @@
+#include <cograph_recognition/CographRecognition.hpp>
+#include <list>
+#include <set>
+#include <bits/stdc++.h>
```

Comment:
Which line does need this include? Please remove it.

Reply:
fixed